examples/java/src/main/java/org/apache/beam/examples/complete/AutoComplete.java

```diff
@@ -479,8 +477,7 @@ public static void main(String[] args) throws IOException {
     if (options.getOutputToDatastore()) {
       toWrite
       .apply(ParDo.named("FormatForDatastore").of(new FormatForDatastore(options.getKind())))
-      .apply(DatastoreIO.writeTo(MoreObjects.firstNonNull(
-          options.getOutputDataset(), options.getProject())));
+      .apply(DatastoreIO.writeTo(options.getProject()));
```

Comment:
Yes this is a common and necessary use case.

On Mon, Jun 20, 2016 at 2:35 PM, Vikas Kedigehalli <notifications@github.com
> wrote:

> In
> examples/java/src/main/java/org/apache/beam/examples/complete/AutoComplete.java
> <https://github.com/apache/incubator-beam/pull/499#discussion_r67771180>:
>
> > @@ -479,8 +477,7 @@ public static void main(String[] args) throws IOException {
> >      if (options.getOutputToDatastore()) {
> >        toWrite
> >        .apply(ParDo.named("FormatForDatastore").of(new FormatForDatastore(options.getKind())))
> > -      .apply(DatastoreIO.writeTo(MoreObjects.firstNonNull(
> > -          options.getOutputDataset(), options.getProject())));
> > +      .apply(DatastoreIO.writeTo(options.getProject()));
>
> What if OutputProject is different from the current Project. Are dataflow
> pipelines allowed to work with multiple projects at the same time?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/apache/incubator-beam/pull/499/files/834b87394f5834f7c6475032cafaed89be977ecb#r67771180>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAgIT7OFwqMhdKeVuxt6f85WKhzfxWFuks5qNwevgaJpZM4I5_6o>
> .
>